src/ripple/rpc/handlers/AccountObjects.cpp

```diff
@@ -81,7 +81,7 @@ Json::Value doAccountObjects (RPC::Context& context)
     {
         using filter_types = std::vector <std::pair <std::string, LedgerEntryType>>;
         static
-        beast::static_initializer <filter_types> const types ([]() -> filter_types {
+        filter_types const types ([]() -> filter_types {
```

Comment:
Not part of this PR, but I don't see why there's a lambda here at all. It can be removed.